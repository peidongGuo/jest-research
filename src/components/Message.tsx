import React from "react";
export default function (props) {
  let { content } = props.message;
  return <li className="list-group-item">{content}</li>;
}
