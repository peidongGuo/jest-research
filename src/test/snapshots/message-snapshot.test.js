import renderer from "react-test-renderer";
import Message from "../../components/Message";

describe("快照测试", () => {
  test("render Message component correctly", () => {
    let message = { id: "1", content: "zhufeng" };
    const tree = renderer.create(<Message message={message} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
