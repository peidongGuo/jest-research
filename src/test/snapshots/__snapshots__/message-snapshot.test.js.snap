// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`快照测试 render Message component correctly 1`] = `
<li
  className="list-group-item"
>
  zhufeng
</li>
`;
